syft/tensor.py

```diff
@@ -1327,35 +2720,87 @@ def masked_select(self, mask):
         return masked_select(self, mask)
 
     def eq(self, t):
-        """Returns a new Tensor having boolean True values where an element of the calling tensor is equal to the second Tensor, False otherwise.
-        The second Tensor can be a number or a tensor whose shape is broadcastable with the calling Tensor."""
+        """
+        Checks if two Tensors are equal.
+
+        Returns a new Tensor having boolean True values where an element of the
+        calling tensor is equal to the second Tensor, False otherwise.
+        The second Tensor can be a number or a tensor whose shape is
+        broadcastable with the calling Tensor.
+
+        Parameters
+        ----------
+        t: Tensor
+            Input tensor
+
+        Returns
+        -------
+        Output tensor;
+        """
         if self.encrypted:
             return NotImplemented
         return TensorBase(np.equal(self.data, _ensure_tensorbase(t).data))
 
     def eq_(self, t):
-        """Writes in-place, boolean True values where an element of the calling tensor is equal to the second Tensor, False otherwise.
-        The second Tensor can be a number or a tensor whose shape is broadcastable with the calling Tensor."""
+        """
+        Checks if two Tensors are equal.
+
+        Writes in-place, boolean True values where an element of the calling
+        tensor is equal to the second Tensor, False otherwise.
+        The second Tensor can be a number or a tensor whose shape is
+        broadcastable with the calling Tensor.
+
+        Parameters
+        ----------
+        t: Tensor
+            Input Tensor
+
+        Returns
+        -------
+        Output tensor;
+        """
         if self.encrypted:
             return NotImplemented
         self.data = np.equal(self.data, _ensure_tensorbase(t).data)
         return self
 
-    def mm(self, tensor2):
-        """Performs a matrix multiplication of :attr:`tensor1` and :attr:`tensor2`.
+    def mm(self, tensor):
+        """
+        Performs a matrix multiplication of two Tensors.
 
         If :attr:`tensor1` is a `n x m` Tensor, :attr:`tensor2` is a `m x p` Tensor,
         output will be a `n x p` Tensor.
 
-        Args:
-            tensor1 (Tensor): First Tensor to be multiplied
-            tensor2 (Tensor): Second Tensor to be multiplied"""
+        Parameters
+        ----------
+        tensor: Tensor
+            Second Tensor to be multiplied
 
-        return syft.mm(self, tensor2)
+        Returns
+        -------
+        n x m Output tensor
```

Comment:
n x p output tensor.

Reply:
Should I change it and commit again and pull request again or what how does it work? I am new to Open Source

Reply:
I figured it out, checkout this branch, commit and then push again.

Reply:
Hey, @CT83 The changes look ok, However, I will leave this open so that we can hear what other folks think about it, and if they like it as well, I think we could move forward.

and you can learn the basic git workflow over [here](https://github.com/OpenMined/Docs/blob/master/tutorials/github_contribution_workflow_tutorial.md)

Reply:
Yes, Thank you! (Deleted my previous Reply as I figured it out myself; later)

Reply:
Done!